src/styles/blocks/header.scss

```diff
@@ -38,7 +38,7 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
-    transition-duration: $transiotion-duration;
+    transition-duration: color $transition-duration;
```

Comment:
Wrong prop. Use [transition shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/transition).

Reply:
I hope my code is okay now))